rs/tests/src/execution/system_api_security_test.rs

```diff
@@ -451,21 +185,11 @@ pub fn malicious_intercanister_calls(env: TestEnv) {
 
     let agent = node.build_default_agent();
 
-    block_on(retry_with_msg_async!(
-        "testing malitious intercanister calls",
-        logger,
-        READY_WAIT_TIMEOUT,
-        RETRY_BACKOFF,
-        || {
-            let effective_canister_id = node.effective_canister_id();
-            let canister_b_wasm_clone = canister_b_wasm.clone();
-            let agent_clone = agent.clone();
-            async move {
-                let canister_b =
-                    create_and_install(&agent_clone, effective_canister_id, &canister_b_wasm_clone)
-                        .await;
-
-                let canister_a_wasm = wat::parse_str(format!(
+    block_on(async move {
```

Comment:
The changes in `malicious_intercanister_calls` only revert adding retry_with_msg_async from https://gitlab.com/dfinity-lab/public/ic/-/merge_requests/20257.The changes in malicious_intercanister_calls only revert adding retry_with_msg_async from [!20257 (merged)](https://gitlab.com/dfinity-lab/public/ic/-/merge_requests/20257).